Feature/logged in/full idea

NielsSensei opened the pull request:
Alles is included en getest. Hieronder een voorbeeldje van hoe een volledig idee eruit ziet. (atm nog een semi-css chaos but it fine for functionality)

![image](https://user-images.githubusercontent.com/44059248/57973569-2219de80-79ab-11e9-86e2-85bb8c531dda.png)

I'm calling for a new css-swoop of ideation als ik klaar ben met edit-idea.

NielsSensei committed:
MapField pre lunch

Added FAQ page waar ik uitleg geef hoe je de coordinaten ophaalt met behulp van google maps.

NielsSensei committed:
MapField Pre Xander

XanderVeldeman committed:
Fix for form

now should be a nice row

NielsSensei committed:
Checking PR

Updated View for DeletedIdea

NielsSensei committed:
Merge branch 'feature/logged-in/full-idea' of https://github.com/NielsSensei/TreeCompany_TheSpanishInquisition_IP1_Team13_KdG1819 into feature/logged-in/full-idea

NielsSensei committed:
MapField Done

NielsSensei committed:
Extra Push 4 MapField (idk why)

NielsSensei committed:
Videofield

Pimped the IdeationMedia linking so dat het autoconvert van gewone yt naar embed yt.

De modview ook geupdate voor MapField (vergeten, oopsie)

Functionaliteiten

NielsSensei committed:
ImageField fuck Images

Functionaliteiten

NielsSensei committed:
ImageField Done

Fixed MediaUrl even more

NielsSensei committed:
ClosedField PreLunch

NielsSensei committed:
ClosedField P1 (gotta redo js)

Questionnaire & Ideation got ModuleType

Ook de view van fields nog wat gepimpt.

Functionaliteiten

NielsSensei committed:
Closed Field Done

NielsSensei committed:
Merge branch 'master' into feature/logged-in/full-idea

XanderVeldeman reviewed (changes requested):
Kan je nog een check toevoegen om ervoor te zorgen dat Title required is? Of dit catchen bij het aanmaken van een idea?
![image](https://user-images.githubusercontent.com/25788097/57973902-4fb55680-79b0-11e9-9336-622143e3d3ef.png)

NielsSensei committed:
Xander Review Fixes

ClosedField now is stored proper
Voting fix
Check for empty title among existing checks on empty idea.

XanderVeldeman reviewed (approved):
Geen problemen meer precies, looks good!
